internal/daemon/daemon.go

```diff
@@ -118,15 +118,6 @@ func (d *Daemon) Run(ctx context.Context, listenPort string, stateDir string, so
 		return fmt.Errorf("Failed to initialize directory structure: %w", err)
 	}
 
-	isAlreadyRunning, err := d.os.IsControlSocketPresent()
```

Comment:
I understand we don't require this extra check as later on during `init()` we run [`removeStale()`](https://github.com/canonical/microcluster/blob/main/internal/endpoints/socket.go#L54) which will get rid of the socket if it already exists?

Please also remove `IsControlSocketPresent()` as it now looks to be unused.

Reply:
It's not used now but will be used by  #149